controllers/ListItemController.php

```diff
@@ -17,6 +17,36 @@
 
 	class ListItemController
 	{
+		/**
+			* Swagger doc block
+			*
+			* @OA\Get(
+			* path="list/item/{listID}",
+			* summary="Find list items by list id",
+			* @OA\Parameter(
+			*		name="listID",
+			* 	in= "path",
+			* 	description="Identifier of the list to fetch items from",
+			* 	required=true,
+			* 	type="string",
+			* 	style="form"
+			*	),
+			* @OA\Response(
+			*		response=200,
+      *   description="successful operation",
+			*		@OA\Schema(
+ 			*   	schema = "ListItem",
+ 			*  		properties= {
+			*		  	@OA\Property(property="id", type="string"),
+			* 	}

```

Comment:
This should be moved outside of the `@OA\Response` block as it is it's own property and does not belong there. You can see an example of that [here](https://github.com/gigili/good-food-tracker-api/blob/main/src/Entity/CityEntity.php). 

Since this projects doesn't have custom entities for each table, you can add it's schemas under the `* Swagger doc block` comment